src/ch01-00-getting-started.md

```diff
@@ -1 +1,56 @@
-Explain first section
\ No newline at end of file
+# Getting started
+
+Let's start our journey with Bastion. During it we will see the main features, we have a lot to discover. For that we will follow a theme, we will build a small CRUD application.
+
+In this first chapter, we will talking about:
+- Installing Bastion on your project
+- Classic and timeless: hello, world!
+
+## Installation
+
+Bastion is a library which need to be added as a dependency to your project. For that, you just have to add it as follow to your `Cargo.toml`:
+```rs
+bastion = "0.3.5-alpha"
+```
+
+## Hello, world!
+The most classic of all examples and especially the essential hello, world! Only for you, in Bastion.
+```rs
+fn main() {
```

Comment:
probably the same nits as https://github.com/bastion-rs/bastion/pull/234 ^^

Reply:
Yes, I will change this one, when we will merge the PR on bastion-rs.